src/contracts/IdleRouter.sol

```diff
@@ -152,7 +155,7 @@ contract IdleRouter is OwnableUpgradeable {
 
         emit TokensDeposited(
             msg.sender,
-            _token,
+            address(underlyingToken),
```

Comment:
I think we can just use `tokenAddress` here, although I am not really sure the cast actually costs us anything additional gas wise.

Reply:
We are casting here because the compiler throws a type error